staging/src/k8s.io/apiserver/pkg/util/webhook/webhook.go

```diff
@@ -103,6 +104,14 @@ func WithExponentialBackoff(initialBackoff time.Duration, webhookFn func() error
 	var err error
 	wait.ExponentialBackoff(backoff, func() (bool, error) {
 		err = webhookFn()
+
+		select {
+		case <-ctx.Done():
```

Comment:
It's slightly cleaner to check for `ctx.Err() != nil` here

Reply:
done